### PR TITLE
Feat/show floors

### PR DESCRIPTION
**Pull request by DanielPopOut**

### CODE REVIEW

File: event.tsx [hunk 1]
```diff
@@ -16,3 +17,12 @@ export default function Event({ event }: { event: AppBooking }) {
     </div>
   );
 }
+
+const getEventStyle = (event: AppBooking) => {
+  if (event.floor.includes('1')) {
```
Comment (aidenbull) [inline event.tsx:22]: Is this equivalent to `event.floor == "1"`? What is the content of the floor string?

File: src/services/OfficeRnDService.ts [hunk 2]
```diff
@@ -40,18 +41,30 @@ export class OfficeRnDService {
     return fetchedData;
   };
 
+  getMeetingRoomsWithFloor = async () => {
```
[the remark this answers is not in the document]
Comment (DanielPopOut) [reply src/services/OfficeRnDService.ts]: Yes would be better to extract it into an AppService :-)

File: src/services/OfficeRnDService.ts [hunk 3]
```diff
@@ -41,16 +41,39 @@ export class OfficeRnDService {
     return fetchedData;
   };
 
-  getMeetingRoomsWithFloor = async () => {
+  getEventsWithMeetingRooms = async (
```
[the remark this answers is not in the document]
Comment (DanielPopOut) [reply src/services/OfficeRnDService.ts:44]: Yess to put to the AppService too

File: event.tsx (hunk 1 above)
Comment (DanielPopOut) [reply event.tsx:22]: No here event.floor is a string, includes checks the value '1' is anywhere in the string
== would check that event.floor equals to '1'

**Pull request merged**